src/nidigital/metadata/enums.py

```diff
@@ -194,47 +194,58 @@
             {
                 'name': 'NIDIGITAL_VAL_0',
                 'python_name': 'ZERO',
+                'pretty_name': '0',
                 'value': 0
             },
             {
                 'name': 'NIDIGITAL_VAL_1',
                 'python_name': 'ONE',
+                'pretty_name': '1',
                 'value': 1
             },
             {
                 'name': 'NIDIGITAL_VAL_L',
+                'pretty_name': 'L',
```

Comment:
We should make the `pretty_name` key optional. If specified, use its value in `__str__`, else use `python_name`.

Reply:
I don't follow, `pretty_name ` is optional right now, and we will only generated `__str__` if `pretty_name ` exists. Are you suggesting to always generate `__str__` and just use `python_name` if `pretty_name ` is not there? That will change the output of a lot of other enums beside these 2.

Reply:
> pretty_name is optional right now

Ah ok, So, you can get rid of `pretty_name` key for "L" (and others), right?

> Are you suggesting to always generate __str__ and just use python_name if pretty_name is not there?

No, we don't want that.

Reply:
Got it.